content/articles/build-a-contact-form-with-php/index.md

```diff
@@ -0,0 +1,349 @@
+            color: white;
+            border-radius: 7px;
+            text-align: center;
+        }
+    </style>
+</head>
+
+
+<body>
+
+    <main class="container">
+        <h4>Contact</h4>
+
+        <hr>
+
+        <div class="row">
+            <div class="col s12 l5">
+                <span class="bold">Get in touch with me via email</span>
+            </div>
+            <div class="col s12 l5 offset-l2">
+
+                <div class="success"><?php echo $success ?></div>
+                <div class="error"><?php echo $error ?></div>
+                <form action="index.php" method="post">
+
+                    <div class="input-field">
+                        <i class="material-icons prefix">account_circle</i>
+                        <input type="text" name="name" id="name" value="<?php echo htmlspecialchars($name) ?>">
+
+                        <label for="name">Your name*</label>
+                        <div class="error"><?php echo $errors["name"] ?></div>
+                    </div>
+                    <div class="input-field">
+                        <i class="material-icons prefix">email</i>
+                        <input type="email" name="email" id="email" value="<?php echo htmlspecialchars($email) ?>">
+                        <label for="email">Your email*</label>
+                        <div class="error"><?php echo $errors["email"] ?></div>
+                    </div>
+                    <div class="input-field">
+                        <i class="material-icons prefix">mode_edit</i>
+                        <textarea id="message" class="materialize-textarea" name="message"><?php echo htmlspecialchars($message) ?></textarea>
+                        <label for="message">Your Message*</label>
+                        <div class="error"><?php echo $errors["message"] ?></div>
+                    </div>
+                    <div class="input-field center">
+                        <input type="submit" class="btn-large z-depth-0" name="submit" id="submit" value="Send"></input>
+                    </div>
+
+                </form>
+
+
+            </div>
+        </div>
+
+
+    </main>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+
+</body>
+
+</html>
+```
+
+**What's happening here**
+
+1. We created a basic form with the **POST** method which ultimately sends the request back to this page.
+
+> To learn more about PHP forms, check out this awesome [article](https://www.section.io/engineering-education/working-with-forms-in-php/).
+
+2. We are also using [materializeCSS](https://materializecss.com/about.html) for some default styling.
+3. And finally we added some PHP variables within our HTML page to echo out either the error messages or the value that you inputted inside your form. Note that we haven't created these variables yet inside our PHP script so it will produce some errors if you refresh your browser.
```

Comment:
```suggestion
2. We are also using [materializeCSS](https://materializecss.com/about.html) for some default styling.
3. We added some PHP variables within our HTML page to echo out either the error messages or the value that was entered in the form. Note that we haven't created these variables yet inside our PHP script so it will produce some errors if you refresh your browser.
```